control_panel_api/tools.py

```diff
@@ -1,13 +1,44 @@
 import secrets
+from collections import UserDict
 
 from django.conf import settings
+from django.utils.functional import cached_property
 from rest_framework.exceptions import APIException
 
 from control_panel_api.helm import Helm
+from control_panel_api.utils import sanitize_environment_variable
+
+
+class HelmToolDeployMixin:
+
+    @cached_property
+    def helm(self):
+        return Helm()
+
+
+class Auth0ClientConfigMixin:
+
+    def _get_auth_client_config(self, key):
+        setting_key = sanitize_environment_variable(
+            f'{self.name}_AUTH_CLIENT_{key}')
+        return getattr(settings, setting_key.upper())
```

Comment:
I may be wrong but you don't need `upper()` here as it's done inside `utils.sanitize_environment_variable()`